gateway/src/main/java/com/hurence/webapiservice/http/grafana/GrafanaApiImpl.java

```diff
@@ -116,6 +127,92 @@ at initialization (did not successfully reproduced this in a unit test).//TODO
                 }).subscribe();
     }
 
+    @Override
+    public void export(RoutingContext context) {
+        final long startRequest = System.currentTimeMillis();
+        final JsonObject requestBody;
+        final QueryRequestParam request;
+        try {
+            requestBody = context.getBodyAsJson();
+            /*
+                When declaring QueryRequestParser as a static variable, There is a problem parsing parallel requests
+                at initialization (did not successfully reproduced this in a unit test).//TODO
+             */
+            request = new QueryRequestParser().parseRequest(requestBody);
+        } catch (Exception ex) {
+            LOGGER.error("error parsing request", ex);
+            context.response().setStatusCode(BAD_REQUEST);
+            context.response().setStatusMessage(ex.getMessage());
+            context.response().putHeader("Content-Type", "application/json");
+            context.response().end();
+            return;
+        }
+        try {
+            int maxDataPoints = requestBody.getInteger(MAX_POINT_REQUEST_FIELD);
```

Comment:
Why using requestBody ??? You already parsed the request so you should just use "request" (adding a convenient method if necessary)

Reply:
done